service/src/main/java/com/epam/esm/service/locale/TranslatorLocale.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.esm.service.locale;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.i18n.LocaleContextHolder;
+import org.springframework.context.support.ResourceBundleMessageSource;
+
+import java.util.Locale;
+
+@Configuration
+public class TranslatorLocale {
```

Comment:
LocaleTranslator if you choose such combination

Reply:
Thank you so much for this advice. (fixed)